src/test/scala/io/iohk/ethereum/mining/BlockGeneratorSpec.scala

```diff
@@ -24,7 +24,8 @@ import scala.concurrent.duration.FiniteDuration
 
 class BlockGeneratorSpec extends FlatSpec with Matchers with PropertyChecks with Logger {
 
-  "BlockGenerator" should "generate correct block with empty transactions" in new TestSetup {
+  // fixme: block used in this test does not pass PoW validation
+  ignore should "generate correct block with empty transactions" in new TestSetup {
```

Comment:
If I understand correctly these tests validate a fully mined block. Does this mean, it's not just these tests, but actually our mining is broken?

Reply:
we do not have mining (yet). These blocks were generated using ethminer and I think something's wrong with them (possibly becuase they were mined on priv network). What I noticed is that if I force validator to use epoch 1 instead of 0 (which should be used for block with num=1) they work.

Reply:
I've mined these blocks again with ethminer, they now work as expected